shell.nix

```diff
@@ -0,0 +1,10 @@
+{ nixpkgs ? import <nixpkgs> { } }:
+
+with nixpkgs;
+
+let  mkEnv = { stdenv, python27, pythonPackages, openssl }:
+  stdenv.mkDerivation {
+    name = "mantl-env";
+    buildInputs = [ python27 pythonPackages.pyyaml openssl terraform packer ansible2 openssh ];
```

Comment:
@avnik Good point, I guess `git` isn't operationally necessary. But we're including some dev tools here (packer) so I don't think it would be out of place either. Up to you!